backend/src/match/match.service.ts

```diff
@@ -1,24 +1,36 @@
 import { Injectable } from "@nestjs/common";
+import { nanoid } from "nanoid";
+import { InjectPinoLogger, PinoLogger } from "nestjs-pino";
 
 import { RedisService } from "src/redis/redis.service";
+import { UserService } from "src/user/user.service";
 
 interface Match {
-  // TODO: Create a room ID with password
+  roomId: string;
   result: { userId: number; socketId: string }[];
+  userNames: string[];
```

Comment:
Instead of returning the user names here, could we have the frontend send a REST query for it instead based on the `userIds` returned?

Reply:
For now the userIds will be displayed on the waiting page. I will do the query for username in another PR.